test/devices_list_test.rb

```diff
@@ -58,6 +58,14 @@
         )
       end
 
+      it "filters by nil values" do
```

Comment:
`filters by missing values` sounds good, as it really search for missing value and not for value that are nil.